ui-cypress/test-module/run.sh

```diff
@@ -24,5 +24,21 @@ ps aux | grep Xvfb
 # disable color output when running Cypress
 export NO_COLOR=1
 #export ELECTRON_EXTRA_LAUNCH_ARGS=--remote-debugging-port=9222
+
+# setup proxy environment variables
+if [ -n "${PROXY_HOST:-}" ]; then
+  if [ -n "${PROXY_HTTPS_PORT:-}" ]; then
+      export HTTP_PROXY="https://${PROXY_HOST}:${PROXY_HTTPS_PORT}"
+      export NODE_EXTRA_CA_CERTS=${PROXY_CA_PATH}
```

Comment:
we need the certificates for both ways
The proxy does MITM termination and responds with a temporary certificate for that